ui/page/tickets/list_page.go

```diff
@@ -71,11 +69,66 @@ func (pg *ListPage) ID() string {
 }
 
 func (pg *ListPage) OnResume() {
+	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
+	pg.wallets = pg.WL.SortedWalletList()
+	components.CreateOrUpdateWalletDropDown(pg.Load, &pg.walletDropDown, pg.wallets)
+	pg.listenForTxNotifications()
+	pg.fetchTickets()
+}
+
+func (pg *ListPage) listenForTxNotifications() {
+	go func() {
+		for {
+			var notification interface{}
+
+			select {
+			case notification = <-pg.Receiver.NotificationsUpdate:
+			case <-pg.ctx.Done():
+				return
+			}
+
+			switch n := notification.(type) {
+			case wallet.NewTransaction:
+				selectedWallet := pg.wallets[pg.walletDropDown.SelectedIndex()]
+				if selectedWallet.ID == n.Transaction.WalletID {
+					pg.fetchTickets()
+					pg.RefreshWindow()
+				}
+			}
+		}
+	}()
+}
+
+func (pg *ListPage) fetchTickets() {
+	var txFilter int32
+	switch pg.ticketTypeDropDown.Selected() {
+	case "All":
+		txFilter = dcrlibwallet.TxFilterStaking
+	case "Immature":
+		txFilter = dcrlibwallet.TxFilterImmature
+	case "Live":
+		txFilter = dcrlibwallet.TxFilterLive
+	case "Voted":
+		txFilter = dcrlibwallet.TxFilterVoted
+	case "Expired":
+		txFilter = dcrlibwallet.TxFilterExpired
+	case "Revoked":
+		txFilter = dcrlibwallet.TxFilterRevoked
+	default:
```

Comment:
Use `pg.ticketTypeDropDown.SelectedIndex()` because the value of `Selected()` can change if another language is used.